### PR TITLE
fix #657

### LINKED ISSUE #657: C:\Windows\SysWOW64\notepad.exe to open file   2end network drive mount shows as "Disconnected Network Drive"
# Environment
 * Windows version: Windows 10 Enterprise N x64
 * Processor architecture: x64
 * Dokany version: master(20180130)
 * Library type : Dokany
 
# Check List
  - [x] I checked my issue [doesn't exist yet](https://github.com/dokan-dev/dokany/issues?utf8=%E2%9C%93&q=is%3Aissue)
  - [x] My issue is valid with mirror default sample and not specific to my user-mode driver implementation
  - [x] I can always reproduce the issue with the provided description below.
  - [x] I have updated Dokany to the [latest version](https://github.com/dokan-dev/dokany/releases) and have reboot my computer after.
  - [x] I tested one of the last [snapshot](https://github.com/dokan-dev/dokany/wiki/Build#user-snapshot) from appveyor CI

# Description
on  Windows 10 Enterprise N x64 
copy dokannp1.dll to  %WINDIR%\system32 and change the registy by the [#656](https://github.com/dokan-dev/dokany/issues/656)
use mirror
mirror.exe /r c:\xx /l M /n /c
mirror.exe /r c:\xx /l T  /n /c
in explorer both drives is ok
but use 32bit notepad.exe which in C:\Windows\SysWOW64   to open file only the first drive is ok.the others show as "Disconnected Network Drive"

2end_drive

### PR DESCRIPTION
**Pull request by HanixNicolas**
I have tested it on windows 10 x64 and windows7 x86 .it works.

### CODE REVIEW

**Comment by Liryna** [comment]
Thank you @HanixNicolas 🚀 
Can you also rebase the history into one commit also ? and I will merge just after.

File: dokan_np/dokannp.c [hunk 1]
```diff
@@ -562,8 +561,23 @@ DWORD APIENTRY NPEnumResource(__in HANDLE Enum, __in LPDWORD Count,
     return WN_NO_MORE_ENTRIES;
   }
 
+  DWORD processId = GetCurrentProcessId();//��ǰ���id
```
Comment (Rondom) [inline dokan_np/dokannp.c]: What does that comment mean?

Comment (HanixNicolas) [reply dokan_np/dokannp.c]: Hi @Rondom ，its a comment in Chinese,I remove it.please ignore it.

**Comment by Liryna** [comment]
Hi @HanixNicolas ,

Can you just rebase all this commit into one to keep the history clean ? ❤️

**Comment by HanixNicolas** [comment]
Hi @Liryna ，I try rebase it，but failed，can you help me to keep the history clean?

**Comment by Liryna** [comment]
Hi @HanixNicolas ,

I have added your fix of to the master #659
https://github.com/dokan-dev/dokany/commit/3ff765496625f3300f667e21ab0e6f969093a777

For #657 we need to see how to act properly not to break ABI

**Comment by Liryna** [comment]
Hi @HanixNicolas ,

The #657 has been included in the master https://github.com/dokan-dev/dokany/commit/3c53dddb99cf523c1b00a82b8fbc9ce37b8dbfd9
Thanks a lot for the both fix and your report 👍

**Pull request closed without merging**